Dwolla.Client/DwollaClient.cs

```diff
@@ -50,6 +51,26 @@ public class DwollaClient : IDwollaClient
         public async Task<RestResponse<TRes>> PostAsync<TReq, TRes>(Uri uri, TReq content, Headers headers) =>
             await SendAsync<TRes>(CreatePostRequest(uri, content, headers));
 
+        public async Task<RestResponse> DeleteAsync<TReq>(Uri uri, TReq content, Headers headers) =>
+            await SendAsync(CreateDeleteRequest(uri, content, headers));

```

Comment:
Couldn't you just do `await SendAsync<object>(CreateDeleteRequest(uri, content, headers));` and avoid adding the non-templated methods?

Reply:
Yeah, this works. I'll remove the non-generic methods.